products/vmware-cloud-director.md

```diff
@@ -0,0 +1,42 @@
+---
+title: VMware Cloud Director
+category: server-app
+tags: vmware
+iconSlug: vmware
+permalink: /vmware-vcd
+alternate_urls:
+-   /vmwarevcd
+releasePolicyLink: https://support.broadcom.com/group/ecx/productlifecycle
```

Comment:
Is there a public page describing the support policy ? If not this link should be moved to the description explaining an a Broadcom account  is needed.